Sources/DreamsViewController.swift

```diff
@@ -25,18 +25,22 @@ public class DreamsView: UIView {
     internal var webService: WebServiceType = WebService()
     internal var dreams = Dreams.shared
 
-    public override init(frame: CGRect) {
-        super.init(frame: frame)
+    public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
```

Comment:
We can also try custom init instead, very popular approach is:

```swift
   init(x: Something) {
        self.x= x
        super.init(nibName: nil, bundle: nil)
    }

    required init?(coder: NSCoder) {
        fatalError("init(coder:) is not supported")
    }
```

Reply:
👍 Yes, this makes sense to do when we inject the dependencies